<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-70</x>
      <y>-30</y>
      <width>551</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnIniciar{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #efa114, stop:1 #fccc7c);
	color:#000000;
	border-radius:5px;
}

QPushButton#btnIniciar:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #7c6e59, stop:1 #FFFFFF);

}

QPushButton#btnIniciar:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:: #7c6e59;
}




QPushButton#Facebook{
	background-color: rgba(0,0,0,0);
	color:rgba(85,98,112,255);
}

QPushButton#Facebook:hover{
	color:rgba(131,96,53,255);
}

QPushButton#Facebook:pressed{
	padding-left:5px;
	padding-top:5px;
	color:rgba(91,88,53,255);
}






QPushButton#Instagram{
	background-color: rgba(0,0,0,0);
	color:rgba(85,98,112,255);
}

QPushButton#Instagram:hover{
	color:rgba(131,96,53,255);
}

QPushButton#Instagram:pressed{
	padding-left:5px;
	padding-top:5px;
	color:rgba(91,88,53,255);
}






QPushButton#WhatsApp{
	background-color: rgba(0,0,0,0);
	color:rgba(85,98,112,255);
}

QPushButton#WhatsApp:hover{
	color:rgba(131,96,53,255);
}

QPushButton#WhatsApp:pressed{
	padding-left:5px;
	padding-top:5px;
	color:rgba(91,88,53,255);
}



QPushButton#Web{
	background-color: rgba(0,0,0,0);
	color:rgba(85,98,112,255);
}

QPushButton#Web:hover{
	color:rgba(131,96,53,255);
}

QPushButton#Web:pressed{
	padding-left:5px;
	padding-top:5px;
	color:rgba(91,88,53,255);
}











</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>49</x>
       <y>30</y>
       <width>271</width>
       <height>521</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/imagenes/colegio.jpg);
border-top-left-radius:50px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>69</x>
       <y>30</y>
       <width>271</width>
       <height>391</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0, 0, 0, 80);
border-image: url(:/newPrefix/marohuma.jpg);

border-bottom-right-radius:50px;
border-top-left-radius:50px;
border-bottom-left-radius:50px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>271</width>
       <height>391</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>6</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #e4ded3;
border-bottom-right-radius:50px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>30</y>
       <width>70</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,100);
border-radius:2px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalWidget_2" native="true">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>30</y>
       <width>70</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color:rgba(0,0,0,0);
	color:rgb(255,255,255);
	border-radius:1px;
	font-size:18px;
	font-family:dripicons-v2;
}

QPushButton:hover{
	background-color:rgb(49,48,53)
}

QPushButton#btnCerrar:hover{
	background-color:rgb(232,17,25)
}

QpushButton:pressed{
	padding-top:5px;
 	padding-left:5px;
}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnMinimizar">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string></string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCerrar">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="IniciarSesion" native="true">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>271</width>
       <height>361</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QPushButton" name="btnIniciar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>221</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Iniciar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtPassword">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>221</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgb(166, 108, 12);
color:rgba(0,0,0,240);
padding-bottom:7px;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Contraseña</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtUsuario">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>221</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgb(166, 108, 12);
color:rgba(0,0,0,240);
padding-bottom:7px;</string>
      </property>
      <property name="placeholderText">
       <string>Usuario</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="Facebook">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Social Media Circled</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Î</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Instagram">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Social Media Circled</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Ú</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="WhatsApp">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Social Media Circled</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Õ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Web">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Social Media Circled</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Å</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>181</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgba(0,0,0,200);
</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Iniciar Sesión&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRecuperar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
color: rgb(0, 0, 0);
background-color: rgb(255, 170, 0);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 ,rgb(255, 170, 0) stop:1, rgb(170, 85, 0));
border-radius: 15px;
border-radius: 15px;

}

QPushButton:hover{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 ,rgb(255, 170, 0) stop:1, rgb(170, 85, 0));
border-radius: 15px;
border-radius: 15px;

}

QPushButton#btnIniciar:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:: #7c6e59;
}</string>
      </property>
      <property name="text">
       <string>¿Ha olvidado su contraseña?</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>201</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 128);
border-top-left-radius:50px;

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>201</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/newPrefix/Marohuma_icono.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../admin/UI MAROHUMA/Marohuma icono.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>label_9</zorder>
    <zorder>IniciarSesion</zorder>
    <zorder>label_13</zorder>
    <zorder>label_7</zorder>
    <zorder>horizontalWidget_2</zorder>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Imagenes/QT_imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
